core/templates/dev/head/collection_editor/navbar/collection_editor_navbar_directive.html

```diff
@@ -0,0 +1,91 @@
+<script type="text/ng-template" id="inline/collection_editor_navbar_directive">
+  <ul
+    class="nav navbar-nav oppia-navbar-nav navbar-right ng-cloak"
+    style="margin-left: 20px;">
+    <!-- Editor -->
+    <li
+      ng-class="{'active': getTabStatuses().active === 'main'">
+      <a
+        href="#"
```

Comment:
FYI: When styling HTML code, you don't need to worry about the 80 character limit we have for JS/Python. Feel free to use more space. How you split it up is up to you, though please be consistent. Consider looking at how other Oppia HTML files handle this.